cmd/flag.go

```diff
@@ -37,6 +37,7 @@ func installFlags(flags *pflag.FlagSet, c *provider.Opts) {
 	flags.StringVar(&c.ListenAddress, "addr", provider.DefaultListenAddr, "address to bind for serving requests from the Kubernetes API server")
 	flags.StringVar(&c.MetricsAddr, "metrics-addr", provider.DefaultMetricsAddr, "address to listen for metrics/stats requests")
 	flags.IntVar(&c.PodSyncWorkers, "pod-sync-workers", provider.DefaultPodSyncWorkers, `number of pod synchronization workers`)
+	flags.IntVar(&c.MapRoot, "map-root", 0, "map the root user to this UID")
```

Comment:
The flag name is non-intuitive to me, so I suggest something like `override-root-uid`.

Reply:
done